template/content/server.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+'use strict';
```

Comment:
this module is the entry point; 

`node server.js` starts the application `app.js` is something you can import in your tests.

I can add more "features" to this file.